Content.Server/Physics/Controllers/MoverController.cs

```diff
@@ -98,60 +100,59 @@ private void HandleShuttleMovement(ShuttleControllerComponent mover)
 
         protected override void HandleFootsteps(IMoverComponent mover, IMobMoverComponent mobMover)
         {
+            if (!mover.Owner.HasTag("FootstepSound")) return;
```

Comment:
Is this tag prototype valid, and does anything have it?

Reply:
Footsteps still play + it was in the old code too (except it was gating it right before the sound is played rather than upfront).